go.mod

```diff
@@ -9,3 +9,5 @@ require (
 	github.com/modern-go/reflect2 v1.0.1 // indirect
 	github.com/stretchr/testify v1.3.0 // indirect
 )
+
+go 1.13
```

Comment:
Unexpected-- if we're making this the start of v2 shouldn't the top line in this file change?

Reply:
Go keeps inserting this in the file ... and so I keep committing it by accident, lovely ...